src/cargo/util/toml/mod.rs

```diff
@@ -1496,8 +1497,8 @@ impl TomlManifest {
     ) -> CargoResult<(Manifest, Vec<PathBuf>)> {
         fn get_ws(
             config: &Config,
-            resolved_path: PathBuf,
-            workspace_config: WorkspaceConfig,
+            resolved_path: &PathBuf,
```

Comment:
Nit: generally you should accept `&Path` instead of `&PathBuf`

Reply:
I can change it to that, if you want